modules/services/yabai/default.nix

```diff
@@ -0,0 +1,107 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.yabai;
+
+  toYabaiConfig = opts:
+    concatStringsSep "\n" (mapAttrsToList
+      (p: v: "yabai -m config ${p} ${toString v}") opts);
+
+in
+
+{
+  options = {
+    services.yabai = with types; {
+      enable = mkOption {
+        type = bool;
+        default = false;
+        description = "Whether to enable the yabai window manager.";
+      };
+
+      package = mkOption {
+        type = path;
+        default = pkgs.yabai;
```

Comment:
This was only added recently to nixpkgs, this default shouldn't be included until 20.09 has been released for a while so people using the stable channel don't run into issues.

Reply:
Fair enough. So this option should simply be removed and default to the available package in the channels

Reply:
Also wondering how this would affect the test approach used here:  
https://github.com/LnL7/nix-darwin/pull/196/files#diff-e566c25b87e15708d8368b82febe145cR11

Reply:
Just means that the option is required. The main reason for overriding that even when there's a default is to reduce the build closure. I don't want to build all sorts of packages just to run these tests and if the package upstream breaks it shouldn't impact the test results either.

Reply:
Ahh, sure okay. I understand 👍 Alright, I'll remove the default.  
Thanks for explaining the test approach, too. That's a nice trick :)